### PR TITLE
US5428 rails 6

### PR DESCRIPTION
**Pull request by RealNobody**
https://rally1.rallydev.com/#/18313249162d/iterationstatus?detail=%2Fuserstory%2F609382265655
https://github.com/CardTapp/mortgagemapp/pull/7559
https://github.com/CardTapp/mongoidable/pull/169
https://github.com/CardTapp/tappforce_engine/pull/276

### CODE REVIEW

File: spec/dummy/config/application.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@
 require "active_model/railtie"
 require "active_job/railtie"
 # require "active_record/railtie"
-require "active_storage/engine"
+# require "active_storage/engine"
```
Comment (jaurand) [inline spec/dummy/config/application.rb:10]: Cant his just be deleted?

File: spec/dummy/config/environments/test.rb [hunk 2]
```diff
@@ -30,8 +30,8 @@
   # Disable request forgery protection in test environment.
   config.action_controller.allow_forgery_protection = false
 
-  # Store uploaded files on the local file system in a temporary directory
-  config.active_storage.service = :test
+  # # Store uploaded files on the local file system in a temporary directory
+  # config.active_storage.service = :test
```
Comment (jaurand) [inline spec/dummy/config/environments/test.rb:34]: ?

File: lib/mongoidable/version.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Mongoidable
-  VERSION = "0.2.0"
+  VERSION = "0.2.1"
```
Comment (jaurand) [inline lib/mongoidable/version.rb:4]: Should this be a larger version bump given the dependency is a major upgrade?

File: spec/dummy/config/application.rb (hunk 1 above)
Comment (RealNobody) [reply spec/dummy/config/application.rb:10]: Yes.  We commented out the other options, so I did the same with this one.

File: spec/dummy/config/environments/test.rb (hunk 2 above)
Comment (RealNobody) [reply spec/dummy/config/environments/test.rb:34]: We aren't using active_storage, so we don't need this configuration

File: lib/mongoidable/version.rb (hunk 3 above)
Comment (RealNobody) [reply lib/mongoidable/version.rb]: OK

**Pull request merged**